### PR TITLE
Paid unpaid redux

### PR DESCRIPTION
**Pull request by jefflembeck**
Make that thing work with ajax.

Error-handling needs to be added still. Let's do that as part of this, just wanted to keep it up to date :smile:

### CODE REVIEW

File: assets/scripts/switch-submission.js [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+      .fail(function() {})
+  }.bind(this));
+};
+
+module.exports = function() {
+  $(function() {
+    var checkboxes = $(".switch");
+
+    $.each(checkboxes, function(idx, checkbox) {
+      var input = new Switch(checkbox);
```
Comment (aredridel) [inline assets/scripts/switch-submission.js]: Making a whole class, just to instantiate it once, and call a single method on it seems kinda indirect. What's the advantage of this approach?

Comment (jefflembeck) [inline assets/scripts/switch-submission.js]: Some of this is certainly YAGNI, but the structure allows for some easy tracking of state and potential growth if we decide to play with growth. Also, I find it a bit easier to unit test.

Comment (aredridel) [inline assets/scripts/switch-submission.js]: Fair enough!

I've usually taken the approach to put something like this bound to a parent element, and have only one instance no matter how many switches, but this should work nicely.

Comment (jefflembeck) [inline assets/scripts/switch-submission.js]: Yeah for sure. I think that I'm mostly doing this multiple binding because of the amount of forms. That being said, I could put the listener on the table, listen for the target and grab the specific form on that.

I'll think of a way to test that well

**Comment by jefflembeck** [comment]
Last step: make sure all tables reflect updated data.
